src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -560,9 +560,11 @@ export const ShippingRoute: FC<ShippingProps> = props => {
     shippingQuotes.length > 0 &&
     !shippingQuoteId
 
+  // TODO: consider to move this block to a useEffect
   if (useDefaultArtsyShippingQuote) {
     const defaultShippingQuoteId = getDefaultShippingQuoteId(order)
-    setShippingQuoteId(defaultShippingQuoteId)
+    shippingQuoteId !== defaultShippingQuoteId &&
```

Comment:
this is just a control to avoid loops, I've tried moving to a useEffect this block but broke other tests, just flagging for future interactions with this file